yarn.lock

```diff
@@ -6547,6 +6547,10 @@ remark-parse@4.0.0:
     vfile-location "^2.0.0"
     xtend "^4.0.1"
 
+rememo@^3.0.0:
```

Comment:
The only downside of the current setup is that we need to ensure that all npm packages are referenced in `package.json` in the main repository folder.